compiler/src/dmd/mtype.d

```diff
@@ -5851,9 +5851,9 @@ extern (C++) final class TypeEnum : Type
         return sym.getMemtype(loc).size(loc);
     }
 
-    Type memType(const ref Loc loc = Loc.initial)
+    Type memType()
```

Comment:
this is never called with a non-default argument.

Reply:
This is a `final` function, right? Couldn't you just make it `extern(D)`?

Reply:
Correct. Maybe, I would consider the removal of an unused default argument a useful side effect refactoring.